backend/aws_helpers/dynamo_db_utils/execution_db.py

```diff
@@ -31,4 +36,53 @@ class ExecutionDDBUtil(BaseDDBUtil):
 
 def get_execution_table(region:str = AWS_REGION) -> BaseDDBUtil:
     """Retrieves the execution-table of an input region as an instance of ExecutionDDBUtil"""
-    return ExecutionDDBUtil(EXECUTION_TABLE_NAME, region)
\ No newline at end of file
+    return ExecutionDDBUtil(EXECUTION_TABLE_NAME, region)
+
+def getOrCreateUserExecutionsData_(entryData: dict) -> str:
+    """
+    Retrieves an entry from the `execution-table` DynamoDB table given an `execution_id`. If does not exist, create a new entry corresponding to the given user_id.
+
+    @param **kwargs: execution_id and other table attributes to be created to the new entry e.g. user_id, if does not exist
+    @return: A JSON string of the entry retrieved or created from the table
+    """
+    dynamoTable = ExecutionDDBUtil(EXECUTION_TABLE_NAME, AWS_REGION)
```

Comment:
redundant

Reply:
you can replace with `get_execution_table`. We want to sort of enforce singleton here